OpenEdXMobile/src/main/java/org/edx/mobile/base/MainApplication.java

```diff
@@ -75,7 +79,7 @@ private void init() {
 
         // initialize Fabric
         if (config.getFabricConfig().isEnabled() && !BuildConfig.DEBUG) {
-            Fabric.with(this, new CrashlyticsCore(), new Answers());
```

Comment:
@BenjiLee do we need `Answers`? cuz its another analytics platform, and afaik we aren't currently working on it.